node/engine/store/memstore.go

```diff
@@ -125,6 +127,17 @@ func (ms *MemStore) SetObjective(obj protocols.Objective) error {
 	return nil
 }
 
+// SetLastBlockNumSeen
+func (ms *MemStore) SetLastBlockNumSeen(blockNumber uint64) error {
+	ms.lastBlockSeen = blockNumber
+	return nil
+}
+
+// GetLastBlockNumSeen
+func (ms *MemStore) GetLastBlockNumSeen() (uint64, error) {
+	return ms.lastBlockSeen, nil
+}
```

Comment:
Let's tidy this up following @lalexgap's comment on block vs block num. It's a pain but when we fix one store implementation, we should fix the other :-P